app/src/main/java/io/apicurio/registry/ccompat/store/RegistryStorageFacadeImpl.java

```diff
@@ -335,6 +335,12 @@ public RuleConfigurationDto getArtifactRule(String subject, RuleType ruleType) {
         return storage.getArtifactRule(null, subject, ruleType);
     }
 
+    @Override
+    public List<Long> getContentIdsReferencingArtifact(String subject, String versionString) {
+        return parseVersionString(subject, versionString,
+                version -> storage.getContentIdsReferencingArtifact(null, subject, version));
```

Comment:
Actually, when working in the legacyId mode, this method must return the list of global ids, instead of the list of content ids.

Reply:
This is exactly what I meant, except that we should implement a new query to get the relation of globalIds from a contentId set. Let me know if you need/want some help with it.